contrib/opencensus-ext-ocagent/tests/test_utils.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018, OpenCensus Authors
```

Comment:
One caveat about Python unittest, the base file name should be unique.
Consider something like "test_ocagent_utils.py".

Reply:
Done